app_dart/lib/src/request_handling/authentication.dart

```diff
@@ -191,7 +189,7 @@ class AuthenticationProvider {
 
       final String clientId = await config.oauthClientId;
       assert(clientId != null);
-      if (token.audience != clientId) {
+      if (token.audience != clientId && !token.email.endsWith('@google.com')) {
```

Comment:
Can you update the dart doc with the gcloud auth info?

Reply:
done